Broken Dockerfile in deployments.md

Just going though Fresh documentation, I believe just by following the documentation and trying to create the Dockerfile as described in [deployments](https://fresh.deno.dev/docs/concepts/deployment) i get the 

```
✘ [ERROR] Deno.Command is not a constructor [plugin deno]

✘ [ERROR] Deno.Command is not a constructor [plugin deno]

✘ [ERROR] Deno.Command is not a constructor [plugin deno]
```

**Cause**:
It seems that `FROM denoland/deno:1.25.0` is the issue in the Dockerfile

**Proposed Solution**
Bump up Dockerfile version to ^1.32.0

PS: If you verify this issue I can of course create the PR :)

--- a/docs/concepts/deployment.md
+++ b/docs/concepts/deployment.md
@@ -36,7 +36,7 @@ caching **will** cause your project to not function correctly.
 Here is an example `Dockerfile` for a Fresh project:
 
 ```dockerfile
-FROM denoland/deno:1.25.0
+FROM denoland/deno:1.32.0
 
 ARG GIT_REVISION
 ENV DENO_DEPLOYMENT_ID=${GIT_REVISION}

--- a/docs/concepts/deployment.md
+++ b/docs/concepts/deployment.md
@@ -36,7 +36,7 @@ caching **will** cause your project to not function correctly.
 Here is an example `Dockerfile` for a Fresh project:
 
 ```dockerfile
-FROM denoland/deno:1.32.0
+FROM denoland/deno:1.33.2
 
 ARG GIT_REVISION
 ENV DENO_DEPLOYMENT_ID=${GIT_REVISION}

